README.md

```diff
@@ -27,3 +27,13 @@ about the current recursive call.
 Describe your reasoning and the conclusion you've come to. Your reasoning is the
 most important part. You do not need to prove that the invariant is correct. Add
 your answer to this markdown file.
+
+##Answers 
+The time invariant makes sure that before calculating the numberfor index I, the previous values have been calculated and stored becasue they are imporant for the next value 
```

Comment:
So what is the invariant?

Reply:
Is it not just the condition that makes sure that the numbers previous have been calculated? If this is not constant than the idea of the sequence fails.

Reply:
That sounds like it's going in the right direction, but your current phrasing only talks about what the invariant does, not what it is.